src/components/Datatable/Pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+
+import Pagination from './Pagination';
+import { PaginationProps } from './Pagination.types';
+
+export default {
+  title: 'components/Datatable/components/Pagination',
+  component: Pagination,
+} as Meta;
+
+export const Default: Story<PaginationProps> = ({ ...args }) => {
+  const [mockPageIndex, setMockPageIndex] = useState(0);
+  const [mockPageCount, setMockPageCount] = useState(args.pageCount);
+  const [isLoading, setIsLoading] = useState(args.isLoading);
+
+  const gotoPage = (page) => {
```

Comment:
Could you add `handle` prefix to handlers? `handleGoToPage, ...` Then it is easier to recognize what is component prop and what is handler itself